automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -725,6 +729,19 @@ def get_sat_cap_version(product):
     ))
 
 
+def post_upgrade_test_tasks(sat_host):
+    """Run set of tasks for post upgrade tier1 tests"""
```

Comment:
I think, this can be for any tier and not for specific 'tier1'. Needs update in docstring.